src/js/shared/constants.js

```diff
@@ -112,6 +128,55 @@ export const NWI_CONFIG = [
   },
 ];
 
+export const MERGE_DROPDOWN_ITEMS = ["Merge", "Override", "Ignore"];
```

Comment:
Due to my comments above, maybe this should be changed to: `MERGE_DROPDOWN_ITEMS = [{key: 'merge', label: "Merge"}, {key: 'override', value: "Override"}, ...];`